ex1-shai/bootstrap.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#Install docker
+
+sudo yum update -y 
```

Comment:
See my comment about sudo in Liroy's PR

Reply:
I saw,
One option to run "sudo su -" and run all the commend from user root.
second option to add the user to /etc/sudoers file and give to the user root privilege or give to the user access to run only  few commands, 
have another options?

Reply:
invoke the bootstrap.sh script with sudo